bigquery_etl/cli/query.py

```diff
@@ -490,13 +490,13 @@ def _backfill_query(
     backfill_date = backfill_date.strftime("%Y-%m-%d")
     if backfill_date not in exclude:
         if destination_table is None:
-            destination_table = table
+            destination_table = f"{project}.{dataset}.{table}"
 
         if not no_partition:
             destination_table = f"{destination_table}${partition}"
 
         click.echo(
-            f"Run backfill for {project}.{dataset}.{destination_table} "
+            f"Run backfill for {destination_table} "
```

Comment:
Is there a check somewhere that errors if it's not fully qualified?

Reply:
Just seeing this sorry 😅 does this account for the `$partition` part of the destination? Given that the qualified check is after the partition check?